l10n_br_account_payment_order/models/account_move_line.py

```diff
@@ -125,8 +130,15 @@ def _compute_journal_entry_ref(self):
             else:
                 record.journal_entry_ref = "*" + str(record.move_id.id)
 
+    def _get_default_service_type(self):
+        if self.move_id.move_type == "in_invoice":
+            return PAGAMENTO_FORNECEDOR
```

Comment:
E se não for?

Reply:
essa versão alpha do módulo só fizemos implementação para pagamento fornecedores, ai conforme for sendo implementando as outras dá pra ir incrementando a logica ai